config/database.yml

```diff
@@ -5,21 +5,21 @@
 #   gem 'sqlite3'
 #
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: db/development
```

Comment:
Let's use a more descriptive db name. Maybe something like medial_control_env:
(i.e. medial_control_development, medial_control_test and medial_control_production)